frontend/src/views/Game.tsx

```diff
@@ -1,31 +1,68 @@
 import Editor from '@monaco-editor/react';
 import React, { useEffect, useState } from 'react';
 import { useLocation } from 'react-router-dom';
+import styled from 'styled-components';
+import { Problem, getProblems } from '../api/Problem';
 import { Room } from '../api/Room';
+import { Text, ProblemHeaderText } from '../components/core/Text';
+import Header from '../components/navigation/Header';
 
 type LocationState = {
   room: Room,
 }
 
-function GamePage() {
+const FlexContainer = styled.div`
+  display: flex;
+  top: 50px;
+`;
+
+const FlexPanel = styled.div`
+  flex: 1;
+  padding: 1rem;
+  background-color: #e3e3e3;
+  overflow: none;
+  height: 100vh;
+  width: 50vw;
+`;
+
+export default function GamePage() {
```

Comment:
nit: Either keep this and change the other `views` files to this format, or change this to `function GamePage()` and keep the other files as-is. Personally, consistency of structure across files is a personal preference for me, and I think will help with readability moving forward.